CHANGELOG.md

```diff
@@ -3,6 +3,18 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 1.0.8 (2022-01-11)
+
+
+### Bug Fixes
+
+* prevent status-timeline from regenerating new viewport on every render ([de1a90e](https://github.com/awslabs/synchro-charts/commit/de1a90e21b44e4baefef167d0321841e8304d9ba))
+* stop ticker when updating viewport duration ([#108](https://github.com/awslabs/synchro-charts/issues/108)) ([c1c1793](https://github.com/awslabs/synchro-charts/commit/c1c1793fab8918f9e4bfd98313728e42eb5302c9))
```

Comment:
https://github.com/awslabs/synchro-charts/pull/109/files

This was already included in 1.0.7 (below). Not sure why it's also being pulled in here.

Reply:
fixed

Reply:
There seems to be an issue with the `versionup:auto` script that will pull all the previous commit that follow conventional-commit into the changelog.